example/lib/screens/regional_payment_methods/ideal_screen.dart

```diff
@@ -41,7 +42,7 @@ class IdealScreen extends StatelessWidget {
     try {
       await Stripe.instance.confirmPayment(
         clientSecret,
-        PaymentMethodParams.ideal(),
+        PaymentMethodParams.ideal(bankName: kIsWeb ? 'revolut' : null),
```

Comment:
why hardcoded revolution here?

Reply:
It is required for the web to add a bankName for the ideal payment. I used one of the one available (the only one I have XD)